### DISCUSSION TITLE
Ya casi termino mi practica CSS

### DISCUSSION THREAD

**Discussion by sergiolopezloya** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#316] [Show and tell]
La vista de escritorio ya está, ahora sólo optimizar para celular y termino:
https://vaccine.animeplay.us

Que les parece?

**Comment by krlosbyte** [comment]
¿A poco si con el conocimiento adquirido en la semana de CSS deberíamos ser capaces de hacer ese diseño?

**Reply by Alemty** [reply]
Jajaa yo me quedé igual, pero supongo que ellos ya conocen bien todo, uno que empieza de cero nos quedará bien simple. Lo importante es entregarlo.

**Reply by DavidQuinter** [reply]
Hola, yo ando empezando desde cero y si está potente jajaja, pero hay mucho contenido de CSS gratis en internet y de ahí me voy guiando.

**Reply by Julioadriglez** [reply]
yo igual ando en ceros y para hacer lo de CSS me pongo haber videos de YouTube o paginas web

**Reply by sergiolopezloya** [reply]
Hola compañeros, aquí por lo que veo es que tenemos que ser auto didactas en muchos aspectos, yo tengo 16 años haciendo web, así que esto no es complicado para mi, pero enfoquémonos que las semanas siguientes van a estar con mayor reto

**Comment by JuanSalvador73** [comment]
Va muy bien. ¿Como resolviste lo del menú transparente que se solapa con la fotografía principal en responsive? Claro que usaste Z-index, pero al encogerlo para dispositivos móviles este menú se acopla perfectamente al pie de esa misma foro...

**Reply by sergiolopezloya** [reply]
La "section" del encabezado está como "position: absolute", de ahi que se va el resto del contenido debajo de ello. Luego a todo el section le puse "background-color: rgba(255, 255, 255, 0.75)" eso te pone el color RGB (255,255,255) o blanco con una opacidad del 75%

**Reply by JuanSalvador73** [reply]
Gracias por el tip :D

**Comment by hackit92** [comment]
Me parece un excelente resultado, creo que el uso de uikit está muy bien implementado, si se notan pequeñas diferencias respecto al original pero son mínimas, excelente práctica

**Reply by sergiolopezloya** [reply]
Gracias, sobre todo la tipografía, pero procuré hacerlo lo más cercano posible

**Comment by lunasaurio** [comment]
Como lo hiciste? En boostrap o algo así o tu mismo desde cero? es que lo estoy tratando de hacer yo mismo y me esta costando mucho trabajo por que no se como igualar las cosas, estoy buscando consejos

**Reply by sergiolopezloya** [reply]
Espero no llegue tarde, jeje, estoy usando uikit, he usado esa librería desde hace años y está muy completa:
https://getuikit.com

**Comment by gersondlarosa** [comment]
Yo ya termine pero no se ve a si jejejeje duda donde subo mi kata

**Reply by sergiolopezloya** [reply]
Yo tengo mi propio servidor, te recomiendo usar uno gratuito, Amazon AWS te da opciones gratuitas pero se requiere cierto conocimiento de servidor:
https://lightsail.aws.amazon.com
